src/Main.java

```diff
@@ -1,5 +1,154 @@
+import org.w3c.dom.ls.LSOutput;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        // Instantiate game model
+        TicTacToeModel model = new TicTacToeModel();
+
+        // Create game board
+        JFrame frame = new JFrame();
+        frame.setSize(800, 600);
+        frame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
+        frame.setTitle("Tic Tac Toe");
+        frame.setVisible(true);
+
+        JPanel topPanel = new JPanel();
+        frame.add(topPanel, BorderLayout.NORTH);
+        JButton clearBoardButton = new JButton("Clear Board");
+        topPanel.setLayout(new FlowLayout());
+        topPanel.add(clearBoardButton);
+
+        JPanel gameGrid = new JPanel();
+        frame.add(gameGrid, BorderLayout.CENTER);
+        gameGrid.setLayout(new GridLayout(3, 3));
+
```

Comment:
the game should be its own class that extends JFrame...will simplify your code enormously!